examples/disclosure/disclosure-navigation-hybrid.html

```diff
@@ -0,0 +1,376 @@
+              <li>
+                <a href="#mythical-page-content">Apply</a>
+              </li>
+              <li>
+                <a href="#mythical-page-content">Tuition</a>
+              </li>
+              <li>
+                <a href="#mythical-page-content">Sign Up</a>
+              </li>
+              <li>
+                <a href="#mythical-page-content">Visit</a>
+              </li>
+              <li>
+                <a href="#mythical-page-content">Photo Tour</a>
+              </li>
+              <li>
+                <a href="#mythical-page-content">Connect</a>
+              </li>
+            </ul>
+          </li>
+          <li>
+            <a href="#mythical-page-content" class="main-link">Academics</a>
+            <button type="button" aria-expanded="true" aria-controls="id_academics_menu" aria-label="Show Academics section"> </button>
+            <ul id="id_academics_menu">
+              <li>
+                <a href="#mythical-page-content">Colleges &amp; Schools</a>
+              </li>
+              <li>
+                <a href="#mythical-page-content">Programs of Study</a>
+              </li>
+              <li>
+                <a href="#mythical-page-content">Honors Programs</a>
+              </li>
+              <li>
+                <a href="#mythical-page-content">Online Courses</a>
+              </li>
+              <li>
+                <a href="#mythical-page-content">Course Explorer</a>
+              </li>
+              <li>
+                <a href="#mythical-page-content">Register for Class</a>
+              </li>
+              <li>
+                <a href="#mythical-page-content">Academic Calendar</a>
+              </li>
+              <li>
+                <a href="#mythical-page-content">Transcripts</a>
+              </li>
+            </ul>
+          </li>
+        </ul>
+      </nav>
+      <div id="mythical-page-content" class="disclosure-pagecontent" tabindex="-1" role="region" aria-label="Mythical University sample page content">
+        <h3 id="mythical-page-heading">Mythical University</h3>
+        <p>Sample content section. Activating a link above will update and navigate to this region.</p>
+      </div>
+    </div>
+    <div role="separator" id="ex_end_sep" aria-labelledby="ex_end_sep ex_label" aria-label="End of"></div>
+  </section>
+
+  <section>
+    <h2>Accessibility Features</h2>
+    <ol>
+      <li>Since this set of disclosure buttons controls visibility of navigation links for the mythical university web site, the list that contains them is wrapped in a navigation landmark named <q>Mythical University</q>.</li>
+      <li>
+        The semantics of the list structure communicates the hierarchy of the navigation system to assistive technology users.
+        The top-level set of three buttons is in a list of three items, and the set of links controlled by each button is contained within a list nested inside the item with the controlling button.
```

Comment:
This point may need a bit of wordsmithing, since there are now 3 links and 3 buttons, making 6 items in the top-level list.

Reply:
Never mind! Just looked at the DOM, and I see that it's coded as 3 list items, each with link and disclosure button children. Makes sense! Resolving this convo.